homeassistant/components/demo/media_player.py

```diff
@@ -398,6 +407,18 @@ def set_repeat(self, repeat):
         self._repeat = repeat
         self.schedule_update_ha_state()
 
+    async def async_join(self, group_members):
+        """Join `group_members` as a player group with the current player."""
+        self._group_members = [
+            self.entity_id,
+        ] + group_members
+        self.schedule_update_ha_state()
```

Comment:
You're in async context.

```suggestion
        self.async_write_ha_state()
```